client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -67,83 +58,17 @@ export class CheckoutThankYouHeader extends PureComponent {
 			return translate( 'You will receive an email confirmation shortly.' );
 		}
 
-		if ( isDomainRegistration( primaryPurchase ) ) {
-			return preventWidows(
-				translate(
-					'Your new domain {{strong}}%(domainName)s{{/strong}} is ' +
-						'being set up. Your site is doing somersaults in excitement!',
-					{
-						args: { domainName: primaryPurchase.meta },
-						components: { strong: <strong /> },
-					}
-				)
-			);
-		}
-
-		if ( isDomainMapping( primaryPurchase ) ) {
-			if ( primaryPurchase.isRootDomainWithUs ) {
-				return translate(
-					'Your domain {{strong}}%(domain)s{{/strong}} was added to your site. ' +
-						'We have set everything up for you, but it may take a little while to start working.',
-					{
-						args: {
-							domain: primaryPurchase.meta,
-						},
-						components: {
-							strong: <strong />,
-						},
-					}
-				);
-			}
-
+		if ( isDelayedDomainTransfer( primaryPurchase ) ) {
```

Comment:
Removing the basic domain transfer code, but leaving the nested condition for delayed transfers in place, as I believe those don't pass flow through the refactored congrats pages.